### PR TITLE
[Feature] Build Reusable Widget for Infographics Screen

### LINKED ISSUE #111: Build re-usable infographic widget
There are a number of infographic screens in the app.  They all need to

- Display a single pinch to zoom image
- Have either a "x" or "<" button in the left hand corner
~- Have a share button in the top right which brings up the OS share sheet to share the image~
- Have configurable theming.  Either blue, purple or green
- Have a configurable title.

**These are the colours to use in green mode**
Yourself : PPE : Detail 02 - Infographic

**These are the colours to use in purple mode**
Yourself : PPE : Detail 05 - Infographic
(note the grey background)

**These are the colours to use in blue mode**
ICU : General Care
(Yes, I know the picture should have the share icon, not the phone.  You know what to do 😁 )

~I'm not sure if we can do different icons per platform, but if possible on Android we should use the 
`icon_share_android.svg` and on ios `icon_share_ios.svg`~

### PR DESCRIPTION
**Pull request by yimingc2**
Resolved issue #111
## Abstract
This PR will create a reusable infographics page with a zoomable image. This page will be used in multiple places.

- [x] `release-notes.txt` has been updated.

## Changes

- Add photo_view dependency
- Add configurable (image, color and title) infographics page for reuse
- Use different share icons based on platform (Android/iOS)
- Apply close (X) icon to left navigation bar button

## Screenshots
zoom

## Things to note
- Not sure what infographic image should be used so I put Intubation Checklist Image for now
- The reusable infographics page has been applied to the Algorithm page for now. The route will be changed in #112

### CODE REVIEW

File: lib/view/reusable_infographic_page.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    return Scaffold(
+      backgroundColor: backgroundColor,
+      appBar: AppBar(
+        iconTheme: AppStyles.appBarIconTheme,
+        title: Text(
+          title,
+          style: AppStyles.appBarTextStyle,
+          ),
+        actions: <Widget>[
+          IconButton(
```
Comment (maks) [inline lib/view/reusable_infographic_page.dart]: could we please break this out into a separate "platform icon button" widget, which takes a named const that represents the required icon, the constant can then be a map, with each asset path keyed by platform name

File: lib/widget/reusable_zoomable_widget.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: <Widget>[
+            Expanded(
+              child: ClipRect(
+                child: PhotoView(
+                  backgroundDecoration: BoxDecoration(
+                    color: backgroundColor
+                  ),
+                  imageProvider: AssetImage(imageUrl),
+                  customSize: Size(MediaQuery.of(context).size.width - margin * 2, MediaQuery.of(context).size.height - margin * 2),
```
Comment (maks) [inline lib/widget/reusable_zoomable_widget.dart]: could we break this calc into a separate function that returns the `Size` for clarity?

**Review by maks — CHANGES_REQUESTED** [review]
:tada: excellent work @yimingc2 ! Just a few small issues to address.

File: lib/view/reusable_infographic_page.dart (hunk 1 above)
Comment (yimingc2) [reply lib/view/reusable_infographic_page.dart]: Do you mean create a general reusable platform icon button or just the button for sharing? If it's first one, then we will need to pass the action as well.

**Review by maks — APPROVED** [review]
:+1: :100:

**Pull request merged**